src/com/itacademy/eshop/simulations/ManagerSimulation.java

```diff
@@ -21,31 +21,42 @@ public void simulate() {
         findAndRemoveOneProduct();
         changePriceForOneProduct();
         removeAllProductsWithCategory(Category.FOOD);
+        addTwoDescriptions();
     }
 
     private void addThreeNewProducts() {
         /**
          * adds three new products to the shop. One of them should be a book, one should be a Laptop, and one should be a shirt.
          */
-        Product laptop = new Product("Laptop", 1000, Category.ELECTRONICS);
+        shop.addProduct(new Product("The Little Prince", 20, Category.BOOKS));
+        shop.addProduct(new Product("Laptop", 1000, Category.ELECTRONICS));
+        shop.addProduct(new Product("Shirt", 25, Category.CLOTHING));
+    }
+
+    private void addTwoDescriptions() {
+        shop.findProductByName("Fidget Spinner").setDescription("A toy to battle anxiety");
+        shop.findProductByName("Crime and Punishment").setDescription("Good book");
     }
 
     private void findAndRemoveOneProduct() {
         /**
          * manager searcher for a product with the name "Laptop" (the one added a moment before) and removes it from the shop
          */
+        shop.removeProductByName("Laptop");
     }
 
     private void changePriceForOneProduct() {
         /**
          * manager searches for a product with the name "Shirt" (the one added a moment before) and changes its price to 20
          */
+        shop.changePriceForProduct("Shirt", 30);
     }
 
     private void removeAllProductsWithCategory(Category category) {
```

Comment:
even though the task says to remove food you should use the method parameter as an input instead of using a constant of FOOD